readme.md

```diff
@@ -253,6 +253,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Hadoop](https://github.com/youngwookim/awesome-hadoop) - Framework for distributed storage and processing of very large data sets.
 - [Data Engineering](https://github.com/igorbarinov/awesome-data-engineering)
 - [Streaming](https://github.com/manuzhang/awesome-streaming)
+- [Apache Spark](https://github.com/awesome-spark/awesome-spark) - Apache Spark is an unified engine for large-scale data processing.
```

Comment:
Another thing in the actual list is that it'd look better without spaces between a couple of the contents bullet points.